getWeatherForecast.py

```diff
@@ -5,7 +5,7 @@
 
 # Retrieves the OpenWeatherMap API key from the environment variables
 def get_api_key():
-    api_key = os.getenv("OPENWEATHER_API_KEY")
+    api_key = os.getenv("OPENWEATHERMAP_API_KEY")
```

Comment:
Rework your own environment variables, not the code. 

Start > "environment variables" > Edit the system environment variables > Environment Variables